latch/ldata/_transfer/utils.py

```diff
@@ -1,5 +1,91 @@
 import os
-from typing import List
+import time
+from enum import Enum
+from typing import Any, Callable, Dict, Optional
+
+from gql.gql import DocumentNode
+from gql.transport.exceptions import TransportClosed, TransportServerError
+from latch_sdk_gql import JsonValue
+from latch_sdk_gql.execute import execute
+
+from latch_cli import tinyrequests
+
+
+class HTTPMethod(Enum):
```

Comment:
```suggestion
class HTTPMethod(str, Enum):
```

Reply:
actually just use the builtin `http.HTTPMethod`

Reply:
Just using str because HTTPMethod is only available in 3.11